addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -349,7 +373,7 @@ odoo.define('website.tour.form_editor', function (require) {
         {
             content:  "Check if required fields were detected and complete the Subject field",
             extra_trigger:  "form:has(#s_website_form_result.text-danger)" +
-                            ":has(.s_website_form_field:has(label:contains('Your Name')).o_has_error)" +
+                            ":has(.s_website_form_field:has(label:contains('Your Name')):not(.o_has_error))" +
```

Comment:
Mmh, why did this change?

Reply:
Just because I needed to test the "value" attribute/property stuff. So I put a default value on the "Your Name" field (which is auto-fill). Since it has a value, when you submit it is not marked as "error" anymore.  If you prefer I can add a step that empties the content of the field (the default value).

Reply:
Oh. Ok. Well at least the previous label saying ""Try to send empty form" should have been adapted then, that's what confused me :grimacing: I guess whatever is fine here, as long as you either change that label or add a step to empty the form.